client/my-sites/site-settings/render-return-button/style.scss

```diff
@@ -0,0 +1,30 @@
+.render-return-button__container {
+	margin: 0;
+	position: fixed;
+		top: 47px;
+		left: 600px;
+		bottom: 200px;
+
+	.button.is-compact.is-borderless {
```

Comment:
`.render-return-button__container` is being applied directly to the button itself, so this property isn't being applied and should be moved to the same level and the first class.